README.md

```diff
@@ -44,6 +44,23 @@ The available development commands are:
 
 Run tests with `npm run test`
 
+### Configuring the app
+
+The app can be configured by modifying:
+- [Ulaanbaatar config](src/config/ulaanbaatar.js)
+- [National config](src/config/national.js)
+
+These files are commented to describe the available options.
+
+**Note:** If you use a new translation key in one of the configuration files, ensure that you add the key to the translation files [found here](src/i18n). See [Translations](#translations) for more info.
+
+### Translations
+
+This app supports English and Mongolian translations via [angular-translate](https://angular-translate.github.io/). For general information, see their documentation.
+
+If you need to simply update the translation keys, those should be modified [here](src/i18n/). Ensure that you add new keys with proper translations to both the EN and MN js files. The application will need to be redeployed after any translation changes.
```

Comment:
Small preferential bit, but I lean against having links that say "click here" or something like that, since users should be able to see it's a link and would be better with the link saying what it pointed to over having to infer that from context. [Some people feel pretty passionate about it](http://www.uxforthemasses.com/click-here/), lol